test/UnitWSDClient.hpp

```diff
@@ -16,16 +16,30 @@
 #include <string>
 #include <vector>
 
+/// Send a command message to WSD from a UnitWSDClient instance on the given connection.
+#define WSD_CMD_BY_CONNECTION_INDEX(INDEX, MSG)                                                    \
```

Comment:
I think I already commented on this elsewhere, I wonder if this could be a function instead of a macro.

Reply:
Yes. This patch should've been included in the previous PR you reviewed, but for some reason I hadn't fixed up the relevant commit with this one. I'll revisit this macro at some point and clean it up (convert to function).